portfolio/src/main/webapp/projects/commentart/scripts/commentsmanager.js

```diff
@@ -0,0 +1,14 @@
+class CommentsManager {
+  /** @param {Array<Comment>} comments All comments that need to be managed */
+  constructor(comments) {
+    this.comments = comments;
+    this.commentsElement = document.getElementById('comments-placeholder');
+    // TODO: ensure that commentsElement exists
+  }
+
+  addComment(replyJson) {
+    const newComment = Comment.toComment(replyJson);
+    this.comments.push(newComment);
+    this.commentsElement.appendChild(newComment.toElement);
```

Comment:
What's the purpose for CommentsManager to hold a reference of commentsElement?

Reply:
Yup! And to summarise the result. I have not added that interface/class for that though.